config.yaml

```diff
@@ -55,14 +55,14 @@ docusaurus:
         - IO:
             name: IO
             location: /build/embed_reveal/IO
-        - Application-Interaction:
+        - Application Interaction:
             name: Application-Interaction
             location: /build/embed_reveal/Application-Interaction
       - Lab:
-        - Software-Stack:
-          - Introduction:
-              location: content/chapters/software-stack/lab
-              name: content/introduction
+        - Setting up the Lab environment:
+            location: content/misc
+            name: lab-setup
```

Comment:
> Are you sure the path to lab-setup in config.yaml is incorrect? It seems correct to me, based on how the other sections are set.

`location` + `name` determine the path to the file. I think one example would solve this:

Software Stack: `locaction = content/chapters/software-stack/lab` and `name = content/overview`. If you look for the overview file's path in Software Stack is `content/chapters/software-stack/lab/content/overview.md`.

Using the same logic for lab-env: `location = content/misc` and `name = lab-setup` leads to the path `content/misc/lab-setup.md`, which is our file.